src/current/cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -62,45 +65,43 @@ All CockroachDB {{ site.data.products.cloud }} organizations get 50M RUs and 10
 
 <section class="filter-content" markdown="1" data-scope="free">
 
-1. Select the **Start for free** option.
+1. On the **Capacity** page, select the **Start for free** option.
 
     {{site.data.alerts.callout_info}}
     This will only be available if you haven't already created a free CockroachDB {{ site.data.products.serverless }} cluster or set up billing information.
     {{site.data.alerts.end}}
 
+1. Click **Next: Finalize**.
+
 </section>
 
 <section class="filter-content" markdown="1" data-scope="paid">
 
```

Comment:
Can you double check that the Free and Paid sections are being displayed as expected?

Reply:
Verified.